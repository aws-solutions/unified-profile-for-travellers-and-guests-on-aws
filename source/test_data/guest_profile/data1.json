{"modelVersion":"1.0","id":"5090100806","lastUpdatedOn":"2022-12-08T01:41:14.409620679Z","createdOn":"2022-10-08T01:41:14.409620679Z","lastUpdatedBy":"Dimitri Wilkinson","createdBy":"Jameson Monahan","emails":[],"phones":[{"type":"home","number":"7729212283","primary":false,"countryCode":8},{"type":"business","number":"1-795-866-3552","primary":false,"countryCode":93}],"addresses":[{"type":"mailing","line1":"8628 Flat port","line2":"more content line 2","line3":"more content line 3","line4":"more content line 4","city":"San Francisco","state":{"code":"","name":""},"province":{"code":"AB","name":"Alberta"},"postalCode":"95346","country":{"code":"NU","name":""},"primary":false},{"type":"shipping","line1":"469 Drive haven","line2":"more content line 2","line3":"more content line 3","line4":"more content line 4","city":"Baltimore","state":{"code":"","name":""},"province":{"code":"ON","name":"Ontario"},"postalCode":"29066","country":{"code":"WF","name":""},"primary":false},{"type":"mailing","line1":"693 Gardens mouth","line2":"more content line 2","line3":"more content line 3","line4":"more content line 4","city":"Nashville-Davidson","state":{"code":"WA","name":"Delaware"},"province":{"code":"","name":""},"postalCode":"98776","country":{"code":"FK","name":""},"primary":false}],"honorific":"Sir","firstName":"Joany","middleName":"Briana","lastName":"Lindgren","gender":"other","pronoun":"he","dateOfBirth":"1997-03-19","language":{"code":"ik","name":""},"nationality":{"code":"CO","name":""},"jobTitle":"Developer","parentCompany":"EMC","loyaltyPrograms":[{"id":"9474425433","programName":"club","points":465729.0,"pointUnit":"reward points","pointsToNextLevel":1049.0,"level":"platinium","joined":"2022-10-08T01:41:14.409620679Z"}]}