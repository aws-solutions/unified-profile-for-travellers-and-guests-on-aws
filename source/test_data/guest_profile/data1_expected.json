{"guest_profile_recs": [{"object_type": "pax_profile", "model_version": "1.0", "last_updated": "2022-12-08T01:41:14.409620679Z", "created_on": "2022-10-08T01:41:14.409620679Z", "last_updated_by": "Dimitri Wilkinson", "created_by": "Jameson Monahan", "honorific": "Sir", "first_name": "Joany", "middle_name": "Briana", "last_name": "Lindgren", "gender": "other", "pronoun": "he", "date_of_birth": "1997-03-19", "job_title": "Developer", "company": "EMC", "nationality_code": "CO", "nationality_name": "", "language_code": "ik", "language_name": "", "email_business": "", "email": "", "email_type": "", "phone": "", "phone_home": "7729212283", "phone_mobile": "", "phone_business": "", "phone_type": "home", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_postal_code": "", "address_billing_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "8628 Flat port", "address_mailing_line2": "more content line 2", "address_mailing_line3": "more content line 3", "address_mailing_line4": "more content line 4", "address_mailing_city": "San Francisco", "address_mailing_state_province": "", "address_mailing_postal_code": "95346", "address_mailing_country": "NU", "address_type": "mailing", "traveller_id": "5090100806", "last_updated_partition": "2022/12/08/01/41/14/409620679Z"}], "common_email_recs": [], "common_phone_recs": [{"object_type": "phone_history", "model_version": "1.0", "last_updated": "2022-12-08T01:41:14.409620679Z", "last_updated_by": "Dimitri Wilkinson", "number": "7729212283", "country_code": 8, "type": "home", "traveller_id": "5090100806", "last_updated_partition": "2022/12/08/01/41/14/409620679Z"}, {"object_type": "phone_history", "model_version": "1.0", "last_updated": "2022-12-08T01:41:14.409620679Z", "last_updated_by": "Dimitri Wilkinson", "number": "1-795-866-3552", "country_code": 93, "type": "business", "traveller_id": "5090100806", "last_updated_partition": "2022/12/08/01/41/14/409620679Z"}], "hotel_loyalty_recs": [{"object_type": "air_loyalty", "model_version": "1.0", "last_updated": "2022-12-08T01:41:14.409620679Z", "last_updated_by": "Dimitri Wilkinson", "id": "9474425433", "program_name": "club", "points": 465729.0, "units": "reward points", "points_to_next_level": 1049.0, "level": "platinium", "joined": "2022-10-08T01:41:14.409620679Z", "traveller_id": "5090100806", "last_updated_partition": "2022/12/08/01/41/14/409620679Z"}]}